src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -51,34 +51,59 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
 
     ngOnInit() {
         this.subscription = this.route.params.subscribe(params => {
+            const didExerciseChange = this.exerciseId === parseInt(params['exerciseId'], 10);
```

Comment:
Shouldn't be `!==` here? I mean, if `this.exerciseId` is the same of the param, I think there is no change :thinking:

Reply:
yeah you are right 😅